Makefile

```diff
@@ -70,9 +70,9 @@ endif
 run-benchmark:
 	@time python -m memory_profiler bin/run_benchmark
 
-test: test-code flake8
+test: setup-data-dir test-code flake8
```

Comment:
Ah ok -- I think it might be good to not download ntuples by default when running "make setup", perhaps we could have "make setup-test" or something which includes them?

Reply:
Simpler might be to add `|| true` after each xrdcp command.  This will at least stop Make dying if the files can't be downloaded.  In `make test` we can add a target that checks and fails if the files haven't been downloaded correctly.

Reply:
|| true would help. However, depending on the size of the test files, do we *need* to download them when doing make setup?